### PR TITLE
Allow client to increase settings save frequency

### PR DESCRIPTION
**Pull request by jherico**

### CODE REVIEW

**Comment by qaAdamSmith82** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13283/HighFidelity-Beta-PR14078-fb1bdee.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13283/HighFidelity-Beta-PR14078-fb1bdee.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13283/HighFidelity-Beta-Interface-PR14078-fb1bdee.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13283/HighFidelity-Beta-PR14078-fb1bdee.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13283/HighFidelity-Beta-Interface-PR14078-fb1bdee.exe

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -318,6 +318,12 @@ static const QString DISABLE_WATCHDOG_FLAG{ "HIFI_DISABLE_WATCHDOG" };
 static bool DISABLE_WATCHDOG = nsightActive() || QProcessEnvironment::systemEnvironment().contains(DISABLE_WATCHDOG_FLAG);
 #endif
 
+static const QString SAVE_SETTINGS_INTERVAL_VAR{ "HIFI_SAVE_SETTINGS_INTERVAL_MS" };
+static const uint32_t DEFAULT_SAVE_SETTINGS_INTERVAL_MS = 10 * MSECS_PER_SECOND;
+static uint32_t SAVE_SETTINGS_INTERVAL_MS = QProcessEnvironment::systemEnvironment().contains(SAVE_SETTINGS_INTERVAL_VAR) 
+    ? QProcessEnvironment::systemEnvironment().value(SAVE_SETTINGS_INTERVAL_VAR).toInt()
+    : DEFAULT_SAVE_SETTINGS_INTERVAL_MS;
```
[the remark this answers is not in the document]
Comment (jherico) [reply interface/src/Application.cpp]: My concern is that setting it to 0 would fire continuously and have potential negative effects with the event queue.  Might be unwarranted, I don't know.

**Pull request closed without merging**